contact_stiffness: -1.0
contact_damping: -1.0
joint_damping: 1.0
lateral_friction: 1.0
spinning_friction: 0.01
rolling_friction: 0.01
resistution: 0.5
order: 5
cutoff: 10